src/madness/world/worldrmi.cc

```diff
@@ -165,6 +165,12 @@ namespace madness {
 
             post_pending_huge_msg();
 
+#if !(defined(HAVE_INTEL_TBB) || defined(HAVE_PARSEC))
+	    // Since this thread never waits or tries to run a task we
+	    // must manually flush the thread-local prebuffer that
+	    // aggregates task submission.
+	    ThreadPool::instance()->flush_prebuf();
```

Comment:
tabs-vs-spaces debates aside (https://thenewstack.io/spaces-vs-tabs-a-20-year-debate-and-now-this-what-the-hell-is-wrong-with-go/) I think most code in MADNESS does not use tabs ... looks like a tab here and elsewhere?

Reply:
Dunno where the tabs came from ... just untabified the whole file.